rfcs/0098-community-team.md

```diff
@@ -0,0 +1,190 @@
+
+The above are some suggested remedies, but these should not be taken as prescriptive. The full space of social actions is open to team members as potential tools for resolving disputes. Team members are expected to use their authority only in ways that embody justice and that are consistent with the community's values, as formalized in the code of conduct and statement of values. Team members are expected to recognize the power dynamic that they are part of, and to be proactive and explicit in making sure community members know when they're speaking in an official capacity and when they're speaking in a personal capacity. Team members are not expected to be impartial in the sense of pretending not to have personal beliefs or goals, but are expected to hold the community's values above their own, in their official actions.
+
+The community team will hold its members accountable to these ideals.
+
+## Code of Conduct and Statement of Values
+[values]: #values
+
+Following the adoption of this RFC, the code of conduct and statement of values will be displayed on a dedicated page on the NixOS website. They will read as follows, unless amended by a future RFC.
+
+The Nix community recognizes that its members come from many walks of life, and that it is natural for us to disagree on some things. Nonetheless, we are all committed to certain values, to allow us to work together and build awesome technologies.
+
+Community members are expected to follow these values in their interactions in all official Nix social spaces, including websites, chat rooms, bug threads, and conferences. We have a community team who will set and enforce social norms, to make that easier for everyone. faCommunity members are expected to follow the community team's requests.
+
+If you have opinions on the community team's work, get involved! It's a rotating team, and we want to develop the leadership potential in all community members.
+
+Our values are as follows.
+
+**Collaboration** — We all want to make the best technologies we can. We work together on our common goals, with open discussion, and without petty infighting or focusing on blame. We are all, always, teaching each other to be better.
+
+**Communication** — When we encounter problems, we focus on solving them, not on fighting each other. When we disagree on technical subjects we follow appropriate processes to escalate them to technical leadership. We try to avoid and de-escalate personal conflicts, but when they are unavoidable we seek mediation.
+
+**Empathy** — We try to see perspectives other than our own, to treat everyone with kindness and empathy, to move past mistakes. When we criticise each other we try to do so graciously. We respect each other's different backgrounds and viewpoints, and we respect the effort everyone puts into the community.
```

Comment:
What I find good in Nix* community is exaclty that somehow collaboration somewhat works even when the perception difference is beyond any realistic ability to empathise. If there are still things that are useful to everyone if done together, they cam happen. I think stressing kindness and empathy over respect and keeping in mind sets an expectation of too much common values.

Reply:
I think we're saying very similar things, but in different ways. That word "empathy" can mean different things to different people. With that said, I do think that making an *attempt* to understand should be part of this. It won't always succeed, but it's important to make the effort.

I think it's also important, when reading all these values but this one especially, to remember that this is both something we as community members are both *expected to pay attention to* and *can expect other people to pay attention to* when interacting with us. It's an obligation that goes in both directions. To anyone who feels that the empathy aspect of this is burdensome, remember that you will also benefit from it.

Reply:
I know I will not benefit from _empathy_ because people are too bad at empathy to perceptions (and while I am normal on major-demographic measurements, I do have relatively weird perceptions w.r.t. interaction with tools, which are of course much more relevant in the context of technical collaboration devoted to tool development).

Understanding and respecting each other's preferences is way more coarse, more intellectual, and so can actually happen (and does happen, and should continue to).

Reply:
I hear that. I'm honestly a very strange person by any objective metric, so I feel like I know where you're coming from, although I don't mean to suggest that that's the same as your experience.

We do talk about respect in here, as you noted, so it seems like what you're asking for is already part of the proposal?

Reply:
My experience are made much more pleasant by built-in tendency to internally perceive some things with condescension (I am not saying it is a _good_ trait, it is just a _comfortable_ one — I am grateful if people call me out if it leaks into communication, it shouldn't unless things are so broken I don't care anymore — and, I honestly can say this condescension doesn't apply even internally to what specifically you say, here or elsewhere).

I think even the mention of respect here is applied to the wrong thing. I neither respect nor disrespect other people's feelings — these are not a thing that can be respected, internal perceptions are outside moral. I try to understand and respect their resulting preferences, and look for a solution that is an improvement to the status quo under the (represented) sets of preferences, or failing that search for a reasonable trade.

Backgrounds, feelings, viewpoints — these are not really reliably interpersonally communicable. I call the operationally expressible part «preferences» (I can say I prefer solution A to solution B, sometimes I can explain «why» well enough that other people guess correctly which of C and D I prefer), maybe another word should be used. I think this is the part where calling for understanding is not pushing towards monoculture along some axis. Understanding feelings benefits too much from having similar feelings even initially.

Reply:
@7c6f434c Would a concept like "forgiveness" or "understanding" be closer?

Reply:
Definitely not forgiveness. Understanding means a ton of things… It looks like the good part is something like collaboration in face of differences.

Reply:
"To see perspectives", maybe?

(That's what the draft already says, but I'd like to call attention to the words again in light of the discussion.)

Reply:
I obviously cannot see your persective. I try to recognise (and take into account) your preferences, though.

Reply:
I don't understand why that's obvious. I am trying to see yours, and I don't think I'm completely failing. Why shouldn't it be a mutual effort?

Reply:
Globally, not just in this conversation? I have no reason to believe you are not completely failing. It isn't well definable of course, as perspectives and ideas of perspectives are not really comparable across people who do not think in very close ways. It's preferences and arguments that we can and should exchange.

Reply:
Yes, globally, although I don't recall talking to you before (please refresh my memory if I've failed to recognize you from elsewhere).

I agree that it's not possible to define with mathematical precision, but I think the community team's process as we've described it in the "rules vs. principles" section will be able to operationalize it.

I do want to exchange preferences and arguments as well, to the extent that that helps me understand your perspective, include you, accommodate your needs, etc. Actually, writing this I think that "accommodation" is the value that captures what you're asking for. I'd be supportive of adding wording to it that makes that more explicit, if you feel that what's already there isn't enough.

Reply:
I am not sure we had long direct exchanges, we were just participating in the same long threads from time to time. Of course in such setting actually getting a detailed impression of perspectives is not reasonable to expect by some of the more optimistic standards than mine.

But in general what stands behind my argumens is not fully recoverable from the arguments themselves (obviously), and I expect from experience that trying to transfer pieces of perspectives is waste of eveyone involved's time in my case.

I think that negotiating actions in the context of irreconcilable value difference is the closest to empathy that still makes sense. Accomodation currently seems to be about taking into account some needs specific to minorities; acknowledgement of value difference is a more global situation.

Reply:
… and by now I am pretty convinced that my priors about empathy not working about somewhat dissimilar things has been confirmed in this discussion, too.

Reply:
That's all quite fair. I hear you saying that you've seen nothing to convince you that I'm able to empathize with you, and I accept that criticism. I won't say "I'm sorry", because I think you're just stating something that you think it's better if we both acknowledge, rather than asking for an apology. I think that's pretty reasonable of you and I'm glad we've been able to talk about it.

Reply:
Yes, nobody should be sorry for being on one side of a gap too large to be able to empathise across.

My confirmation that empathy is impossible is basically how some of your replies to arguments are not to the point of the arguments the first time (and then sometimes the second time too), even though it is not any kind of very novel argument. I know you have to had seen similar ones, and your answers reveal you do not understand the viewpoint of the other side. Which is fine, it means we who make the argument word it sloppily and will try to do better — but also shows that one cannot run things on empathy without some kind of preexisting similarity of thinking.

That's why I think empathy as defined in the dictionary just doesn't work reliably enough, pretending that it does just leads to confusion, and so both a better headline word and a slightly different «expanded explanation» are necessary.